orientation/exercises/10_TEST_ANIMALS.md

```diff
@@ -19,8 +19,9 @@ As a team, we'll be building unit test coverage for all the functionality of the
 
 Write test cases to verify the I/O of the following methods of `Animal` and `Dog`.
 
+<!-- TODO: IN my code, I never create an instance of Animal. Is it necessary? -->
 1. In the test class' `setUpClass()` method, create an instance of `Animal` and `Dog`.
-1. Animal object has the correct `name` property.
+1. Animal object has the correct `name` property.  <!-- What is correct name property?-->
```

Comment:
When the student creates an `Animal` instance, it should have a `name` property. Perhaps clearify that in the previous step?

Reply:
👍